README.md

```diff
@@ -45,3 +45,21 @@ Docs for resources and their arguments can be found [here](docs). For a thorough
 - [increasing IPs in active deployment](https://github.com/threefoldtech/terraform-provider-grid/issues/15)
 - [introducing new nodes to kuberentes deployment](https://github.com/threefoldtech/terraform-provider-grid/issues/13)
 - [multiple deployments on the same node](https://github.com/threefoldtech/terraform-provider-grid/issues/11)
+
```

Comment:
The `Troublehooting` section is deprecated, the common errors that were common are checked against in the plugin at runtime.

Reply:
The deprecated `Troubleshooting` section was removed.